noark-extraction-validator/src/main/java/com/documaster/validator/config/properties/InternalProperties.java

```diff
@@ -41,8 +44,13 @@ public InternalProperties(List<String> propertyFiles) throws IOException, Config
 
 			try (InputStream is = getClass().getClassLoader().getResourceAsStream(file)) {
```

Comment:
We no longer need the stream it looks like. We can get rid of the `throws IOException` declaration for the constructors as well.

Reply:
Changed w/ https://github.com/documaster/noark-extraction-validator/pull/66/commits/b58f7551f05889b80d1e9f6a529710525dfa8533

Reply:
We still need to remove the throws IOException declaration in the `Noark5Properties` class constructor

Reply:
Removed w/ https://github.com/documaster/noark-extraction-validator/pull/66/commits/5817337587bc9bf4934ee24bdfaedff585796dad